server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -623,6 +626,7 @@ func (e *EventChannel) FinalizeInvocation(iid string) error {
 	}
 
 	ctx, cancel := background.ExtendContextForFinalization(e.ctx, 10*time.Second)
+	ctx = log.EnrichContext(ctx, log.InvocationIDKey, iid)
```

Comment:
Can you move this to OpenChannel, that way the invocation_id will be available through out the entire lifecycle.

Reply:
Done